lib/algo/DMD.C

```diff
@@ -185,6 +187,8 @@ DMD::constructDMD(const Matrix* f_snapshots,
     }
 
     std::cout << "Using " << d_k << " basis vectors out of " << num_singular_vectors << "." << std::endl;
+    CSVDatabase* db_CSVOutput(new CSVDatabase);
+    db_CSVOutput->putDoubleArray("SV.csv", d_factorizer->S, num_singular_vectors);
```

Comment:
I think there maybe a problem if we have multiple dmd objects within the same simulation, like in Laghos, and they all write to "SV.csv"

Reply:
I think what would work is having a separate write function for singular values, and having the user input a string path.

Reply:
Thanks. I will take care of this later.

Reply:
A new function `DMD::summary(outputPath)` is introduced